config.template.json

```diff
@@ -16,6 +16,11 @@
         ],
         "woisping_limit": 7200,
         "woisping_threshold": 2,
+        "faulenzerping_ignored_role_ids": [
```

Comment:
Finde wir sollten eher whitelisten oder hat das einen besonderen Grund?

Reply:
War eigentlich für "@everyone" gedacht. Aber durch die anderen Änderungen ist das fast schon überflüssig geworden.

Bin mir nicht sicher mit Whitelisting.

Reply:
Mein Gedanke dabei ist, dass ich so theoretisch die ganzen Sprachen-Rollen ignorieren muss. Sonst kann ich ja einfach bei jeder normalen Umfrage faulenzerping C machen und den halben Server pingen.

Reply:
Gegen Whitelisting würde sprechen, dass man es immer nur verwenden kann, wenn man die Conifg vom Bot geupdated hat. Das wär dann immer relativ umständlich.

Vielleicht kann man die Whitelist auch in die DB schieben und sie von Mods pflegen lassen. Da würde mir aber der Scope zu groß.

Also Rollen-Whitelist in die Config machen?

Reply:
Würde ich vorschlagen. Wir haben ja nur ne handvoll Rollen, wo das Feature wirklich relevant ist.

Könnten vielleicht in einem späteren Schritt (teilweise) die gesamte Config als K/V-Store in die DB schieben und dann ein Modcommand hinzufügen, mit dem man die bearbeiten kann.